src/release/issue-triaging.md

```diff
@@ -75,25 +75,48 @@ Triaging them the same way as `needs-triage` is also useful.
 There are many different labels that can be applied to issues.
 
 - `needs-triage`: Signals that an issue is new and needs initial triage
-- `T-*`: Specifies the team or teams that this issue is relevant to, for example compiler, types or libs
-- `WG-*`: Specifies the working groups that this issue is relevant to, for example WG-debugging.
-- `C-*`: Specifies the category of the label, for example a bug, tracking issue or discussion
-    - `A-diagnostics` issues usually don't have any C label.
+- [`T-*`]: Specifies the **team** or teams that this issue is relevant to. For example `T-compiler`, `T-types` or `T-libs`.
+- [`WG-*`]: Specifies the **working groups** that this issue is relevant to, for example `WG-debugging`.
+- [`PG-*`]: Specifies the **project groups** that this issue is relevant to, for example the `PG-exploit-mitigations`.
+- [`C-*`]: Specifies the **category** of the label, for example a bug, tracking issue or discussion
+    - `A-diagnostics` issues usually don't have any `C-*` label.
     - Also note `C-optimization` for missed compiler optimizations.
-- `O-*`: For platform-specific issues, specifies the platform (architecture or operating system). For example macos, aarch64, windows
-- `A-*`: The areas that the issue is relevant to, for example linkage, patterns, diagnostics
-- `F-*`: When the issue concerns a specific (usually unstable) feature
+- [`O-*`]: For platform-specific issues, specifies the **platform** (architecture or operating system). For example `O-macos`, `O-aarch64`, `O-windows`.
```

Comment:
I think we also have like operating-system + compiler, e.g. `O-windows-mvsc`, so maybe call this "target" or "target families"? It's also possible for an issue to be relevant to multiple target families. Don't know what the least confusing terminology here is.

Reply:
Lol, I've just noticed that I'm highlighting **p**latform which doesn't make sense :facepalm:

Reply:
Makes sense. I've changed the terminology to *(compile) target (family)* while highlighting the *platform (arch or OS)* as the most notable instance of it.